collect_app/src/test/java/org/odk/collect/android/utilities/FileUtilsTest.java

```diff
@@ -35,4 +39,79 @@ public void mediaDirNameIsCorrect() {
         assertEquals(expected, FileUtils.constructMediaPath("sample-file.123"));
         assertEquals(expected, FileUtils.constructMediaPath("sample-file.docx"));
     }
+
+    @Test public void identifiesFormMetadata_formSimpleForm() throws IOException {
+        String simpleForm = "<?xml version=\"1.0\"?>\n" +
+                "<h:html xmlns=\"http://www.w3.org/2002/xforms\"\n" +
+                "        xmlns:h=\"http://www.w3.org/1999/xhtml\"\n" +
+                "        xmlns:orx=\"http://openrosa.org/xforms\">\n" +
+                "    <h:head>\n" +
+                "        <h:title>My Survey</h:title>\n" +
+                "        <model>\n" +
+                "            <instance>\n" +
+                "                <data id=\"mysurvey\">\n" +
+                "                </data>\n" +
+                "            </instance>\n" +
+                "        </model>\n" +
+                "    </h:head>\n" +
+                "    <h:body>\n" +
+                "\n" +
+                "    </h:body>\n" +
+                "</h:html>";
+        File temp = File.createTempFile("simple_form", ".xml");
+        temp.deleteOnExit();
+
+        BufferedWriter out = new BufferedWriter(new FileWriter(temp));
+        out.write(simpleForm);
+        out.close();
+
+        HashMap<String, String> metadataFromFormDefinition = FileUtils.getMetadataFromFormDefinition(temp);
+
+        assertThat(metadataFromFormDefinition.get(FileUtils.TITLE), is("My Survey"));
+        assertThat(metadataFromFormDefinition.get(FileUtils.FORMID), is("mysurvey"));
+        assertThat(metadataFromFormDefinition.get(FileUtils.VERSION), is(nullValue()));
+        assertThat(metadataFromFormDefinition.get(FileUtils.BASE64_RSA_PUBLIC_KEY), is(nullValue()));
+    }
+
+    @Test public void identifiesFormMetadata_forFormWithSubmission() throws IOException {
```

Comment:
Could this be `getMetadataFromFormDefinition_withSubmission_returnsMetaDataFields()`?